features/security-monitoring.md

```diff
@@ -0,0 +1,28 @@
+# Security Monitoring
+The Private Packagist security monitoring feature analyzes the dependencies of your private packages for known security vulnerabilities.
+Packages get analyzed on every commit and every time a database reports new vulnerabilities.
```

Comment:
Packages are analyzed every time we find a new commit in your package and when a new vulnerability is published in one of the databases.